src/ResourceManager/Consumption/Commands.Consumption/help/Get-AzureRmConsumptionUsageDetail.md

```diff
@@ -38,25 +25,25 @@ The **Get-AzureRmConsumptionUsageDetail** cmdlet gets usage details of the subsc
 ## EXAMPLES
 
 ### Example 1
-```
-PS C:\> Get-AzureRmConsumptionUsageDetail -IncludeMeterDetails -InvoiceName 201704-117283130069214
+```powershell
+PS C:\> Get-AzureRmConsumptionUsageDetail -Expand MeterDetails -Top 10
 ```
 
-Get usage details of the invoice with specified name, and include MeterDetails property in the result.
```

Comment:
Please provide example output for these examples.

Reply:
@maddieclayton Our model of usage detail has 28 output parameters. Can we skip output example to avoid user distraction?